packages/features/lend/src/lend-fixed/hooks/use-totals-data.tsx

```diff
@@ -5,23 +5,20 @@ import {
   FiatSymbols,
   TokenDefinition,
 } from '@notional-finance/core-entities';
-import {
-  useAllMarkets,
-  useMaxSupply,
-} from '@notional-finance/notionable-hooks';
+import { useMaxSupply } from '@notional-finance/notionable-hooks';
+import { useCurrentNetworkStore } from '@notional-finance/notionable';
 
 export const useTotalsData = (
   deposit: TokenDefinition | undefined,
   collateral: TokenDefinition | undefined,
   baseCurrency: FiatKeys
 ) => {
-  const {
-    yields: { fCashLend },
-  } = useAllMarkets(deposit?.network);
+  const currentNetworkStore = useCurrentNetworkStore();
+  const fCashLend = currentNetworkStore.getAllFCashYields();
```

Comment:
Inside of these use-totals-data hooks instead of fetching all of the yields and then filtering, we should call a function that gets exactly the values we need. If you look lower down the component it looks like we are filtering out for the TVL, we can get that value directly from the store instead of doing things the old way.